pkg/controller/userreport/index.go

```diff
@@ -56,6 +58,15 @@ func (c *Controller) HandleIndex() http.Handler {
 		}
 
 		m := controller.TemplateMapFromContext(ctx)
+
+		now := time.Now().UTC()
+		pastDaysDuration := -1 * c.config.IssueConfig().AllowedSymptomAge
+		displayAllowedDays := fmt.Sprintf("%.0f", c.config.IssueConfig().AllowedSymptomAge.Hours()/24.0)
+		m["maxDate"] = now.Format(project.RFC3339Date)
```

Comment:
This was a bug. The HTML was trying to set these values, but they weren't provided. Thus the date picker allowed any possible date.